web/scenes/Onboarding/CreateTeam/page/index.tsx

```diff
@@ -48,10 +47,10 @@ export const CreateTeamPage = async (props: CreateTeamPage) => {
           </LayersIconFrame>
 
           <Typography as="h1" variant={TYPOGRAPHY.H6} className="text-center">
-            {hasMemberships ? "Create a new team" : "Create your first team"}
+            {hasUser ? "Create a new team" : "Create your first team"}
```

Comment:
technically this doesn't check if this is their first team or not just that they exist?

Reply:
can just leave it as create a new team

Reply:
ah nvm I see now that the user isn't created until we create a team